docs/content/preview/yugabyte-platform/troubleshoot/install-upgrade-yp-issues.md

```diff
@@ -64,3 +64,23 @@ do
   ssh $IP firewall-cmd --zone=public --add-port=6379/tcp;
 done
 ```
+
+## Configure load balancer for Helm charts
```

Comment:
"Configure load balancer for YugabyteDB Anywhere Helm charts" would be better because we have helm charts for both YugabyteDB and YugabyteDB Anywhere.

Reply:
This whole Troubleshooting page is under YugabyteDB Anywhere product, so everything described here is only applicable to that product unless stated otherwise.